packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/shippingRadios.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import { mergeClasses } from '../../../classify';
+import LoadingIndicator from '../../LoadingIndicator';
+import RadioGroup from '../../RadioGroup';
+import ShippingRadio from '../../CartPage/PriceAdjustments/ShippingMethods/shippingRadio';
+import defaultClasses from './shippingRadios.css';
+
+const ShippingRadios = props => {
+    const {
+        hasShippingMethods,
+        isLoading,
+        selectedShippingMethod,
+        shippingMethods
+    } = props;
+
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    if (isLoading) {
+        return (
+            <LoadingIndicator>{`Loading Shipping Methods...`}</LoadingIndicator>
+        );
+    }
+
+    if (!hasShippingMethods) {
+        return (
+            <span className={classes.error}>
+                {`Error loading shipping methods.
+                Please ensure a shipping address is set and try again.`}
+            </span>
+        );
+    }
+
+    const shippingRadios = shippingMethods.map(method => {
+        const label = (
+            <ShippingRadio
```

Comment:
It's awesome that we can reuse some of the cart page components. That said, the `CartPage` shipping radios puts the selection price under the text whereas the mocks for the checkout page radios show the price to the right of the text. Personally I prefer we keep them identical on both cart and checkout but it is something we should raise with UX.

cc @soumya-ashok 

[![Image from Gyazo](https://i.gyazo.com/12c01cc4468a5dac4d22004f836dfbe0.png)](https://gyazo.com/12c01cc4468a5dac4d22004f836dfbe0)